styles/nav-bar.css

```diff
@@ -70,6 +71,34 @@
   margin-right: 20px;
 }
 
+.nav-toggle-show {
+  visibility: visible;
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  top: 0;
+  left: 0;
+  background: #fff;
+  width: 100%;
+  height: 100vh;
+  z-index: 1;
+  transition: all 0.5s ease-in-out;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-bottom-left-radius: 50px;
+  border-bottom-right-radius: 50px;
+  justify-content: flex-start;
+  padding-left: 1rem;
+  padding-top: 2rem;
+  gap: 20px;
+}
+
+.nav-item-show {
+  visibility: visible;
+  width: 100%;
+  justify-content: center;
+  margin-top: 2rem;
+}
```

Comment:
- [ ] You did a great job on the functionality but you need to fix the below style issues to match with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=Om4oWoF5BWrjny9M-0) design.
  - Please use the correct color for the close(X) button color `#A7B8B0`. 
  - Please increase the font size for mobile menu options.
  - Please reduce the gap between mobile menu options.

### Your design

![image](https://github.com/waiyankyaw961999/Microverse-portofolio/assets/83803414/ac0e266a-df6d-47c6-96bb-109a508995d4)

### Figma design

![image](https://github.com/waiyankyaw961999/Microverse-portofolio/assets/83803414/0c3ec5d9-d58f-4d97-b82f-3b22fbe9f2e0)